### PR TITLE
Update chart-features-highlighting.md

### PR DESCRIPTION
**Pull request by mddifilippo89**
mdd-seo-chart-features-highlighting

### CODE REVIEW

File: doc/en/components/charts/chart-features-highlighting.md [hunk 1]
```diff
@@ -7,7 +7,58 @@ mentionedTypes: []
...
+</code-view>
+
+<div class="divider--half"></div>
+
+
+## Highlight Layers
+
+The $ProductName$ category chart component can enable three types of highlighting when hovering over items.
+
+1. Series Highlighting will highlight the single data point when the pointer is positioned over it.
```
Comment (HUSSAR-mtrela) [inline doc/en/components/charts/chart-features-highlighting.md:24]: change "the single data point"
to "the single data point represented by a marker or column"

File: doc/en/components/charts/chart-features-highlighting.md [hunk 2]
```diff
@@ -7,7 +7,58 @@ mentionedTypes: []
...
+
+```html
+<igc-category-chart
+    id="chart"
+    width="800px"
+    height="500px"
+    is-series-highlighting-enabled="true"
+    is-category-highlighting-enabled="true"
+    is-item-highlighting-enabled="true" >
+</igc-category-chart>
```
Comment (HUSSAR-mtrela) [inline doc/en/components/charts/chart-features-highlighting.md:63]: in all above snippets, we should not set all highlighting props to true. Set only one property to true and other to false:
```
    isSeriesHighlightingEnabled="false"
    isCategoryHighlightingEnabled="false"
    isItemHighlightingEnabled="true
```

**Review by HUSSAR-mtrela — CHANGES_REQUESTED** [review]

File: doc/en/components/charts/chart-features-highlighting.md (hunk 1 above)
Comment (mddifilippo89) [reply doc/en/components/charts/chart-features-highlighting.md:24]: pending

File: doc/en/components/charts/chart-features-highlighting.md (hunk 2 above)
Comment (mddifilippo89) [reply doc/en/components/charts/chart-features-highlighting.md:63]: changes pending

commented out the snippets for now. Updated the body of text that describes the feature 
eg. 
1. Series Highlighting will highlight the single data point represented by a marker or column when the pointer is positioned over it. **This is enabled by setting the `IsSeriesHighlightingEnabled` property to true.**

Comment (mddifilippo89) [reply doc/en/components/charts/chart-features-highlighting.md:63]: Note, snippets are commented out due to seo purposes until further notice.

Comment (HUSSAR-mtrela) [reply doc/en/components/charts/chart-features-highlighting.md:63]: please fix the code snippets regardless whether they are visible or not.

**Review by HUSSAR-mtrela — CHANGES_REQUESTED** [review]

File: doc/en/components/charts/chart-features-highlighting.md (hunk 2 above)
Comment (mddifilippo89) [reply doc/en/components/charts/chart-features-highlighting.md:63]: pending

**Pull request merged**